src/main/java/com/sprint/mission/discodeit/entity/UserStatus.java

```diff
@@ -0,0 +1,16 @@
+package com.sprint.mission.discodeit.entity;
+
+public enum UserStatus {
+    ACTIVE("온라인"),
+    INACTIVE("오프라인"),
+    AWAY("자리비움"),
+    DELETED("삭제된 유저"),
+    BANNED("차단된 유저"),;
+
+    private String userStatus;
```

Comment:
```suggestion
    private String description;
```
정도가 적당한 네이밍 같아 보입니다.